janusgraph-cql/src/main/java/org/janusgraph/hadoop/formats/cql/CqlBinaryInputFormat.java

```diff
@@ -79,19 +76,5 @@ public void setConf(final Configuration config) {
         ConfigHelper.setInputColumnFamily(config, janusgraphConf.get(CQLConfigOptions.KEYSPACE),
             mrConf.get(JanusGraphHadoopConfiguration.COLUMN_FAMILY_NAME), wideRows);
         log.debug("Set keyspace: {}", janusgraphConf.get(CQLConfigOptions.KEYSPACE));
-
```

Comment:
Everything in this PR is really just copying files from Apache Cassandra to us and updating a dependency, except for these lines that you remove here. Can you give a short explanation of why they can be removed now?

Reply:
We didn't had a test for it. SlicePredicate doesn't exists anymore. If we want this feature again, we should test it.

Reply:
I see, thanks for the explanation!